apps/io-services-cms-webapp/src/main.ts

```diff
@@ -140,6 +141,15 @@ export const serviceReviewCheckerEntryPoint = createReviewCheckerHandler(
   fsmLifecycleClient
 );
 
+export const createRequestReviewLegacyEntryPoint =
+  createRequestReviewLegacyHandler(
+    fsmLifecycleClient,
+    getDao({
+      ...config,
+      REVIEWER_DB_TABLE: config.REVIEWER_LEGACY_DB_TABLE,
```

Comment:
**suggestion (non-blocking):** by doing as suggested below, we could have avoided modifying infra files and app configs
```suggestion
      REVIEWER_DB_TABLE: `${config.REVIEWER_DB_TABLE}_legacy`,
```

Reply:
4a5858e